release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -144,6 +144,77 @@ Netlink messages dropped by OVS kernel module due to netlink socket buffer overf
+
+This feature is only available with supported Container Storage Interface (CSI) drivers.
+
+For more information, see xref:../storage/container_storage_interface/ephemeral-storage-csi-inline.adoc[CSI inline ephemeral volumes].
+
+==== Automatic CSI migration for Microsoft Azure File auto migration is generally available
+Starting with {product-title} 4.8, automatic migration for in-tree volume plugins to their equivalent Container Storage Interface (CSI) drivers became available as a Technology Preview feature. Support for Azure File was provided in this feature in {product-title} 4.10, and {product-title} 4.13 now supports automatic migration for Azure File as generally available. CSI migration for Azure File is now enabled by default and requires no action by an administrator.
+
+This feature automatically translates in-tree objects to their counterpart CSI representations and should be completely transparent to users. Translated objects are not stored on disk, and user data is not migrated.
+
+While storage class referencing to the in-tree storage plugin will continue working, it is recommended that you switch the default storage class to the CSI storage class.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-migration.adoc[CSI Automatic Migration].
+
+==== Automatic CSI migration for VMware vSphere auto migration is generally available
+Starting with {product-title} 4.8, automatic migration for in-tree volume plugins to their equivalent Container Storage Interface (CSI) drivers became available as a Technology Preview feature. Support for vSphere was provided in this feature in {product-title} 4.10, and {product-title} 4.13 now supports automatic migration for vSphere as generally available. CSI migration for vSphere is now enabled by default and requires no action by an administrator.
+
+This feature automatically translates in-tree objects to their counterpart CSI representations and should be completely transparent to users. Translated objects are not stored on disk, and user data is not migrated.
+
+While storage class referencing to the in-tree storage plugin will continue working, it is recommended that you switch the default storage class to the CSI storage class.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-migration.adoc[CSI Automatic Migration].
+
+==== Cross account support for AWS EFS CSI driver is generally available
+Cross account support allows you to have an {product-title} cluster in one Amazon Web Services (AWS) account and mount your file system in another AWS account using the AWS Elastic File System (EFS) Container Storage Interface (CSI) driver.
```

Comment:
s/Cross account support/Cross-account support ??

Reply:
Really not sure about this, so I'm leaving it be.